.gitignore

```diff
@@ -0,0 +1,42 @@
+# This file contains default .gitignore rules. To use it, copy it to .gitignore,
+# and it will cause files like your settings.php and user-uploaded files to be
+# excluded from Git version control. This is a common strategy to avoid
+# accidentally including private information in public repositories and patch
+# files.
+#
+# Because .gitignore can be specific to your site, this file has a different
+# name; updating Drupal core will not override your custom .gitignore file.
+# Ignore core when managing all of a project's dependencies with Composer
+# including Drupal core.
+# core
+# Ignore dependencies that are managed with Composer.
+# Generally you should only ignore the root vendor directory. It's important
+# that core/assets/vendor and any other vendor directories within contrib or
+# custom module, theme, etc., are not ignored unless you purposely do so.
+/vendor/
+# Ignore configuration files that may contain sensitive information.
+web/sites/*/settings*.php
+web/sites/*/services*.yml
+# Ignore paths that contain user-generated content.
+web/sites/*/files
+# sites/*/private
+private/
+# Ignore files and directories generated by Drupal core
+web/sites/*/files/
+web/sites/default/*/files
+web/modules/contrib/
+web/modules/custom/custom_block_module/
+web/modules/custom/movie_directory/
```

Comment:
Why add custom code to .gitgnore?

Reply:
Now have added custom_block_module, earlier had added to .gitignore as It was not in task and was only for practise.